telegraf.conf

```diff
@@ -499,3 +526,10 @@
     domains = ["www.alipay.com","www.wechat.com","www.baidu.com", "www.agora-space.com"]
     port = 53
     timeout = 2
+
+[[inputs.dns_query]]
+    servers = ["10.1.0.1", "10.1.0.2", "10.1.0.4"]
+    network = "udp"
+    domains = ["www.alipay.com","www.wechat.com","www.baidu.com", "www.agora-space.com"]
+    port = 53
+    timeout = 2
```

Comment:
this should have a name tag based on the dns  server.
ex : internal_dns_servers

Reply:
Do you mean:
`[[inputs.dns_query.tags]]

name = "internal_dns_servers"`

Is it possible to put on everything any tags?